File: containers/ruby-2/test-project/hello.rb
#-----------------------------------------------------------------------------------------
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License. See LICENSE in the project root for license information.
#-----------------------------------------------------------------------------------------

def sayHello(name)
  result = "Hello, " + name + "!"
  return result
end
  
  
puts sayHello("Remote Extension Host")
puts sayHello("Local Extension Host")

File: containers/r/test-project/hello.R
#-------------------------------------------------------------------------------------------------------------
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
#-------------------------------------------------------------------------------------------------------------

say_hello <- function(name) {
  message(paste0("Hello, ", name, "!"))
}

say_hello("remote world")
